R/item_upload_files.R

```diff
@@ -7,6 +7,10 @@
 #' @param files A string vector of paths to files to be uploaded
 #' @param scrape_files logical should the files be scraped for metadata? 
 #' If TRUE, sciencebase will attempt to create extensions based on the files.
+#' 
+#' For example, for shapefiles, this will result in a shapefile extension 
+#' to be returned as a facet of the sciencebase item.
```

Comment:
Is there a place where `facets` are defined? I  personally didn't even know what `facets` meant until I started reviewing this and saw the example item with the different facets for different shapefiles.

Reply:
Good question. It's part of the sciencebase item model, which I guess I assume people have some knowledge of if they are programming against sciencebase... Maybe a bad assumption. Do you think it would be worth linking to the main sciencebase documentation? https://my.usgs.gov/confluence/display/sciencebase/ScienceBase+Item+Core+Model#ScienceBaseItemCoreModel-facets(a.k.a.Extensions)

Reply:
I think linking there or even just to a publicly accessible item that has facets would be useful